src/ship/PlayerShipController.h

```diff
@@ -5,6 +5,7 @@
 
 #include "Input.h"
 #include "ShipController.h"
+#include <optional>
```

Comment:
And again, to reiterate my original comment from the first part of the review, I think std::optional is not an optimal fit for the class variable, as its state is handled nicely by the presence of `m_followTarget`.

Reply:
Removed!